user-storage-simple/src/main/java/org/keycloak/quickstart/writeable/PropertyFileUserStorageProvider.java

```diff
@@ -254,7 +254,7 @@ public boolean isValid(RealmModel realm, UserModel user, CredentialInput input)
         UserCredentialModel cred = (UserCredentialModel)input;
         String password = properties.getProperty(user.getUsername());
         if (password == null || UNSET_PASSWORD.equals(password)) return false;
-        return password.equals(cred.getValue());
+        return password.equals(cred.getChallengeResponse());
```

Comment:
Remove this - we need to fix API issues on the server side

Reply:
I sent PR to fix this on server side. If you have a chance to review, it is here https://github.com/keycloak/keycloak/pull/6795

Reply:
Together with @stianst, we agreed to temporarily disable both User Storage modules. I created [KEYCLOAK-13051](https://issues.redhat.com/browse/KEYCLOAK-13051) to re-enable them once this fix is in place.